### ISSUE TITLE
about code (2)

### ISSUE THREAD

**Issue by Alexey-T** [dingo-d/CSS-Table-of-Contents#6]
```
        pos = self.view.sel()[0].begin() # Cursors position
        contents = self.view.substr(sublime.Region(0, pos)) # Contents of the file up to the position of the cursor
        region = self.view.line(self.view.sel()[0]) # Cursor region
        selected = self.view.substr(sublime.Region(self.view.sel()[0].begin(), self.view.sel()[0].end())) # Selected text
        line = self.view.substr(self.view.line(self.view.sel()[0])) # Returns the line contents of the cursor
```

why not to save self.view.sel()[0] to var?

**Comment by dingo-d** [comment]
There are some minor changes like that that I may have missed. I'll fix those in the next release, I still have some changes to add. Thanks for the notice :)

**Comment by dingo-d** [comment]
I've released v1.1.1 with this fixed, and some minor linter issues I've found.
